draft-ietf-taps-arch.md

```diff
@@ -577,6 +581,30 @@ RFC-EDITOR: Please remove this section before publication.
 
 This document has no actions for IANA.
 
+# Operations and Management Considerations
+
+The Transport Services architecture provides a framework for a substantially
+wider interface between the application and transport layers than the existing
+sockets interface, and as such affords more degrees of freedom for configuring
+and operating hosts using it.
+
+From a management perspective, the most important abstraction in the architecture
+is the System Policy, which constrains and influences, at the host
+level, how an implementation will gather Candidate Paths and Protocol Stacks
+and race the candidates when establishing a Connection. As the details of
+System Policy configuration and enforcement are largely platform- and
+implementation- dependent, and do not affect application-level interoperability,
+the Transport Services API {{?I-D.ietf-taps-interface}} does not specify
+an interface for reading or writing System Policy.
+
+The Transport Services API specifies both synchronous and asynchronous methods
+for reporting connection errors to the application, as the application is the most
```

Comment:
```suggestion
for reporting errors to the application, as the application is the most
```

Reply:
I wanted to suggest "Connection" and then understood that these could be more general (hence "connection"), but they can also be related to Messages...  all in all, the word "connection" here is unnecessary and can cause confusion, IMO.

Reply:
hah, i thought this already said "connection errors" 😆. applied.